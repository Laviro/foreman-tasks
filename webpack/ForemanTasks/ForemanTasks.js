import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { LinkContainer } from 'react-router-bootstrap';
import { ButtonGroup, Button } from 'patternfly-react';

import routes from './Routes/ForemanTasksRoutes';
import ForemanTasksRouter from './Routes/ForemanTasksRouter';

const ForemanTasks = () => (
  <BrowserRouter>
    <div>
      <div style={{ paddingTop: '10px', paddingBottom: '10px' }}>
        <ButtonGroup bsSize="large">
          <LinkContainer to={routes.indexTasks.path}>
            <Button bsStyle="link">index-tasks-page</Button>
          </LinkContainer>
          <LinkContainer to={routes.showTask.path.replace(':id', 'some-id')}>
            <Button bsStyle="link">show-task-page</Button>
          </LinkContainer>
        </ButtonGroup>
      </div>
      <ForemanTasksRouter />
    </div>
  </BrowserRouter>
);

export default ForemanTasks;
