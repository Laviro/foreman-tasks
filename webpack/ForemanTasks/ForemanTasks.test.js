import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import ForemanTasks from './ForemanTasks';

const fixtures = {
  'render without Props': {},
};

describe('ForemanTasks', () =>
  testComponentSnapshotsWithFixtures(ForemanTasks, fixtures));
