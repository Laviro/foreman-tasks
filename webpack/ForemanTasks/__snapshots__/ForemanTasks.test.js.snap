// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanTasks render without Props 1`] = `
<BrowserRouter>
  <div>
    <div
      style={
        Object {
          "paddingBottom": "10px",
          "paddingTop": "10px",
        }
      }
    >
      <ButtonGroup
        block={false}
        bsClass="btn-group"
        bsSize="large"
        justified={false}
        vertical={false}
      >
        <LinkContainer
          activeClassName="active"
          exact={false}
          replace={false}
          strict={false}
          to="/foreman_tasks/ex_tasks"
        >
          <Button
            active={false}
            block={false}
            bsClass="btn"
            bsStyle="link"
            disabled={false}
          >
            index-tasks-page
          </Button>
        </LinkContainer>
        <LinkContainer
          activeClassName="active"
          exact={false}
          replace={false}
          strict={false}
          to="/foreman_tasks/ex_tasks/some-id"
        >
          <Button
            active={false}
            block={false}
            bsClass="btn"
            bsStyle="link"
            disabled={false}
          >
            show-task-page
          </Button>
        </LinkContainer>
      </ButtonGroup>
    </div>
    <ForemanTasksRouter />
  </div>
</BrowserRouter>
`;
